src/Client/Suggest.cpp

```diff
@@ -191,7 +183,7 @@ void Suggest::fillWordsFromBlock(const Block & block)
 
     size_t rows = block.rows();
     for (size_t i = 0; i < rows; ++i)
-        words.emplace_back(column.getDataAt(i).toString());
+        new_words.emplace_back(column.getDataAt(i).toString());
```

Comment:
Can we just call `addWords()` here and get rid of necessity of having `new_words` as a persistent variable.

Reply:
Decided to put make new_words local variable, to process words by batches at least.